canonical-web-and-design/charmed-osm.com
# Add new cookie policy

## Done

- Add new cookie policy
- Created new GTM container
- Created new GA view
- Fixed html tag nesting
- Revealed bullets

## QA

- Check out this feature branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8042/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the cookie policy works as expected


## Issue / Card

Fixes #53
